package.json

```diff
@@ -354,6 +355,7 @@
     "d3-cloud": "1.2.5",
     "dedent": "^0.7.0",
     "delete-empty": "^2.0.0",
+    "dompurify": "^2.4.1",
```

Comment:
@ananzh looks like this conflicts with the reporting plugin https://github.com/opensearch-project/reporting/blob/main/dashboards-reports/package.json#L22. Should we create a PR into their repo?